src/Dianoga/Dianoga.csproj

```diff
@@ -84,4 +84,7 @@
 	  <None Remove="Default Config Files\Dianoga.WebP.CDN.config.disabled" />
 	  <None Remove="Dianoga Tools\libwebp\COPYING.txt" />
 	</ItemGroup>
+	<ItemGroup>
+	  <PackageReference Include="System.Threading.Tasks.Dataflow" Version="5.0.0" />

```

Comment:
So looking at this https://www.nuget.org/packages/System.Threading.Tasks.Dataflow/5.0.0 and this https://stackoverflow.com/questions/23245713/is-tpl-dataflow-included-with-either-net-4-5-or-net-4-5-1/23245803 looks like we will need to add this dependency that will flow up. Seems like this will be fine on .NET 4.5+ but I would love it to be tested on Sitecore 8 before I push this out, so will tag this as pre-release when I push the package.